checkov/cdk/checks/python/LambdaFunctionLevelConcurrentExecutionLimit.yaml

```diff
@@ -0,0 +1,18 @@
+metadata:
```

Comment:
Notice that you also can implement the adding reserved_concurrent_executions as following, although I think it's less 'programatic' way, we should here what @tsmithv11 has to say regarding that
```
class LambdaStack(core.Stack):

    def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
        super().__init__(scope, id, **kwargs)

        # Define the Lambda function using the low-level CfnFunction
        function = CfnFunction(self, "ExampleLambdaFunction",
                               runtime="python3.8",
                               handler="index.handler",
                               code=Code.from_asset("lambda").asset_path,  # Directory containing your Lambda code
                               function_name="ExampleLambdaFunction"
                               )

        # Set the reserved concurrency using CfnConcurrency
        reserved_concurrency = core.CfnResource(self, "ExampleLambdaFunctionConcurrency",
                                                type="AWS::Lambda::Concurrency",
                                                properties={
                                                    "FunctionName": function.function_name,
                                                    "ReservedConcurrentExecutions": 100
                                                })

        # Ensure concurrency settings are applied after the function is created
        reserved_concurrency.node.add_dependency(function)


app = core.App()
LambdaStack(app, "LambdaStack")
```